tuneme/static/css/styles.css

```diff
@@ -75,6 +75,7 @@ input[type='submit']:focus, a.button:focus {opacity:0.5;}
         .grey a.button.inverted {color:#828282; background:white; border-color:#828282;}
 
 
+.decoration {text-decoration: none;}
```

Comment:
@saeedmb8 `.step-links a` also, you can add it to `smart.css` if you only want it to apply to the smart version